dev-client/src/components/home/BottomSheet.tsx

```diff
@@ -83,16 +33,30 @@ const SiteListBottomSheet = ({sites, showSiteOnMap}: Props) => {
     navigation.navigate('CREATE_SITE');
   }, [navigation]);
 
+  const {colors} = useTheme();
+  const style = useMemo(() => ({paddingHorizontal: 16}), []);
```

Comment:
question: what is the use of `useMemo` here?

Reply:
it's (supposed) to avoid re-renders, since if you pass in a raw object on each render, it will pass a new object in and cause a re-render (i think)